cmd/server-main.go

```diff
@@ -345,6 +346,16 @@ func configRetriableErrors(err error) bool {
 		errors.Is(err, os.ErrDeadlineExceeded)
 }
 
+func bootstrapTrace(msg string) madmin.TraceInfo {
+	return madmin.TraceInfo{
+		TraceType: madmin.TraceBootstrap,
+		Time:      time.Now().UTC(),
+		NodeName:  globalLocalNodeName,
+		FuncName:  "BOOTSTRAP",
```

Comment:
You can capture the line number and source @vadmeste instead of just a string.

Reply:
> You can capture the line number and source @vadmeste instead of just a string.

just one line or the whole stack trace ?

Reply:
Just the func name+file:line no of the caller.

Reply:
yeah `"BOOTSTRAP: "+getSource(1)`  @vadmeste perhaps works?

Reply:
Oh wait you already have this in message, its not needed.